Words from Pod::Wordlist::hanekomu were merged into Pod::Wordlist
at Pod::Wordlist 1.06; if we have an older version of Pod::Wordlist,
we'll need to use Pod::Wordlist::hanekomu instead.

--- t/author-pod-spell.t
+++ t/author-pod-spell.t
@@ -12,7 +12,7 @@
 
 # generated by Dist::Zilla::Plugin::Test::PodSpelling 2.006001
 use Test::Spelling 0.12;
-use Pod::Wordlist;
+use Pod::Wordlist::hanekomu;
 
 
 add_stopwords(<DATA>);
